docs/fatjar-regressions/fatjar-regressions-v0.36.1-SNAPSHOT.md

```diff
@@ -102,7 +102,7 @@ Here are the instructions for reproducing runs on the MS MARCO V2.1 segmented do
 ```bash
 TOPICS=(msmarco-v2-doc-dev msmarco-v2-doc-dev2 trec2021-dl trec2022-dl trec2023-dl rag24-raggy-dev); for t in "${TOPICS[@]}"
 do
-    java -cp $ANSERINI_JAR io.anserini.search.SearchCollection -index msmarco-v2.1-doc-segmented -topics $t -output $OUTPUT_DIR/run.msmarco-v2.1-doc-segmented.bm25.${t}.txt -threads 16 -bm25 -hits 10000 -selectMaxPassage -selectMaxPassage.delimiter "#" -selectMaxPassage.hits 1000
```

Comment:
actually, `doc-segmented` is correct. The two MS MARCO 2.1 conditions are `doc` and `doc-segmented`.

Reply:
For the current setup, the output file names are of the form `run.collection-name.condition-name.topic-name.txt`. I would modify it such that `collection-name` is `msmarco-v2.1`, `condition-name` in {`doc`, `doc-segmented`}, so the resulting file names would look like, for example:
`run.msmarco-v2.1.doc.trec2021-dl.txt`
`run.msmarco-v2.1.doc-segmented.trec2021-dl.txt`
Would this be desired?

Reply:
yup!